indent/verilog_systemverilog.vim

```diff
@@ -65,16 +65,16 @@ if exists("*GetVerilogSystemVerilogIndent")
   finish
 endif
 
-if exists("g:verilog_disable_indent")
-  let s:verilog_disable_indent = split(g:verilog_disable_indent, ",")
-else
-  let s:verilog_disable_indent = []
-endif
-
 set cpo-=C
 
 function! GetVerilogSystemVerilogIndent()
 
+  if exists("g:verilog_disable_indent")
+    let s:verilog_disable_indent = split(g:verilog_disable_indent, ",")
+  else
+    let s:verilog_disable_indent = []
+  endif
+
```

Comment:
They're not measured but estimates based off when I was trying to squeeze performance when I re-implemented the indent script. With that said there was a large error margin when I was profiling so my numbers may be completely off.